### PR TITLE
feat(dashboard): add helper functions

### PR DESCRIPTION
**Pull request by Gukkey**

### CODE REVIEW

File: src/app/dashboard/helper.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+type InviteCodeReturnJsonFailure = Omit<InviteCodeReturnJsonSuccess, "inviteCode">
+
+export async function generateInviteCode(
+  role: string
+): Promise<InviteCodeReturnJsonSuccess | InviteCodeReturnJsonFailure> {
+  const rString = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+  let inviteCode = ""
+  for (var i = 0; i < 5; i++) {
```
Comment (shiyaamsunder) [inline src/app/dashboard/helper.ts]: use let instead of var.
```suggestion
  for (let i = 0; i < 5; i++) {
```

File: src/app/dashboard/helper.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    inviteCode += rString[Math.floor(Math.random() * rString.length)]
```
Comment (shiyaamsunder) [inline src/app/dashboard/helper.ts:22]: Does this always always pick an index less than the length of the string?

File: src/app/dashboard/helper.ts (hunk 1 above)
Comment (Gukkey) [reply src/app/dashboard/helper.ts]: yeah got it.

File: src/app/dashboard/helper.ts (hunk 2 above)
Comment (Gukkey) [reply src/app/dashboard/helper.ts:22]: Think so, I tried generating around 10 pairs of random strings and never found a badly formed string or an error.

**Review by shiyaamsunder — APPROVED** [review]

**Pull request merged**